challenges/cat-photo-gallery/script.js

```diff
@@ -0,0 +1,36 @@
+window.onload = getRandomCatPicture ()
+
+async function getRandomCatPicture () {
+    await fetch("https://cataas.com/cat?json=true")
+    .then(response => response.json())
+    .then(randomCat => {
+        const image = document.querySelector("img");
+        return image.src = "https://cataas.com" + randomCat.url;
+    }).catch(error => "Error: " + error)
+}
+
+document.onreadystatechange = () => {
+    if (document.readyState === "complete") {
+
+        const checkbox = document.querySelector("#checkbox");
+        const input = document.querySelector("#input");
+        input.disabled = true;
+
+        checkbox.addEventListener("change", function () {
+            !checkbox.checked ? input.disabled = true : input.disabled = false;
```

Comment:
this is a bit wrong way to write ternary operator.
it should be:
```js
input.disabled = checkbox.checked ? true : false;
```

but that would be the same as writing:
```js
input.disabled = checkbox.checked;
```
but what you really want to do here is making a trigger:
```js
checkbox.addEventListener("change", function () {
        checkbox.checked = !checkbox.checked;
        input.disabled = checkbox.checked;
    })
```